kubejs/server_scripts/base/unification/unify_loot.js

```diff
@@ -1,196 +1,80 @@
 // https://github.com/AlmostReliable/lootjs/wiki
 LootJS.modifiers((event) => {
-    // Replace ingots / nuggets / Storage Blocks / Gems in loot chests!
+    // How to use unifyLoot()
+    // tag              -> String - Tag to search for (accepts both tags with and without #)
+    // suffix           -> String - suffix for the EE variant of the item
+    // prefix           -> String - prefix for the EE variant of the item (for example, when you want to unify crushed ores (...:crushed_iron_ore))
+    // ItemStack        -> ItemStack - Original ItemStack that was meant to be dropped
+    // replaceWithRaw   -> Boolean - Should it replace the drop with Raw ore, or return unified version of the item?
+    // count            -> integer - if replaceWithRaw is True, then specifies the amount of raw ore given for each item.
+    function unifyLoot(tag, suffix, ItemStack, prefix, replaceWithRaw, count) {
+        prefix = (typeof prefix == undefined || prefix == null)? "": prefix + "_";
+        replaceWithRaw = (typeof replaceWithRaw == undefined || replaceWithRaw == null)? false: true;
+        count = (typeof count == undefined || count == null)? ItemStack.getCount(): count;
+        // If itemstack already is from EE and replaceWithRaw is false, just return it.
+        if (ItemStack.getId().startsWith('emendatusenigmatica:') && !replaceWithRaw) {return ItemStack;}
+        tag = (tag.startsWith("#")? tag.substring(1): tag) + "/"
+        let iterator = ItemStack.getTags().iterator();
+        while (iterator.hasNext()) {
+            let tagString = iterator.next().location().toString();
+            if (tagString.startsWith(tag)) {
+                let tagSubString = tagString.substring(tag.lastIndexOf("/")+1);
+                if (Item.exists(`emendatusenigmatica:raw_${tagSubString}`) && replaceWithRaw) {
+                    return Item.of(`emendatusenigmatica:raw_${tagSubString}`, count);
+                } else if (Item.exists(`emendatusenigmatica:${tagSubString}_${suffix}`)) {
```

Comment:
No need for `else` when there's a return in the if above

Reply:
👍